src/parkapi_sources/converters/opendata_swiss/models.py

```diff
@@ -0,0 +1,203 @@
+"""
+Copyright 2024 binary butterfly GmbH
+Use of this source code is governed by an MIT-style license that can be found in the LICENSE.txt.
+"""
+
+from datetime import datetime, time, timezone
+from enum import Enum
+from typing import Optional
+
+from validataclass.dataclasses import validataclass
+from validataclass.exceptions import ValidationError
+from validataclass.validators import (
+    DataclassValidator,
+    EnumValidator,
+    IntegerValidator,
+    ListValidator,
+    Noneable,
+    StringValidator,
+    TimeFormat,
+    TimeValidator,
+)
+
+from parkapi_sources.converters.base_converter.pull import GeojsonFeatureGeometryInput
+from parkapi_sources.models import RealtimeParkingSiteInput, StaticParkingSiteInput
+from parkapi_sources.models.enums import ParkAndRideType, ParkingSiteType, PurposeType
+
+
+@validataclass
+class OpenDataSwissAddressInput:
+    addressLine: str = StringValidator()
+    city: str = StringValidator()
+    postalCode: str = StringValidator()
+
+
+class OpenDataSwissCapacityCategoryType(Enum):
+    STANDARD = 'STANDARD'
+    DISABLED_PARKING_SPACE = 'DISABLED_PARKING_SPACE'
+    RESERVABLE_PARKING_SPACE = 'RESERVABLE_PARKING_SPACE'
+    WITH_CHARGING_STATION = 'WITH_CHARGING_STATION'
+
+
+@validataclass
+class OpenDataSwissCapacitiesInput:
+    categoryType: OpenDataSwissCapacityCategoryType = EnumValidator(OpenDataSwissCapacityCategoryType)
+    total: int = IntegerValidator()
+
+
+@validataclass
+class OpenDataSwissAdditionalInformationInput:
+    de: Optional[str] = Noneable(StringValidator(multiline=False, unsafe=True))
+    en: Optional[str] = Noneable(StringValidator(multiline=False, unsafe=True))
+    it: Optional[str] = Noneable(StringValidator(multiline=False, unsafe=True))
+    fr: Optional[str] = Noneable(StringValidator(multiline=False, unsafe=True))
+
+
+class OpenDataSwissOperationTimeDaysOfWeek(Enum):
+    MONDAY = 'MONDAY'
+    TUESDAY = 'TUESDAY'
+    WEDNESDAY = 'WEDNESDAY'
+    THURSDAY = 'THURSDAY'
+    FRIDAY = 'FRIDAY'
+    SATURDAY = 'SATURDAY'
+    SUNDAY = 'SUNDAY'
+
+    def to_osm_opening_day_format(self) -> str:
+        return {
+            self.MONDAY: 'Mo',
+            self.TUESDAY: 'Tu',
+            self.WEDNESDAY: 'We',
+            self.THURSDAY: 'Th',
+            self.FRIDAY: 'Fr',
+            self.SATURDAY: 'Sa',
+            self.SUNDAY: 'Su',
+        }.get(self, None)
```

Comment:
The `None` would mean that you return `None` if there is no key (and the typing would be wrong). As you described all keys, the `None` can just be removed.